Projects/Server/Items/Item.cs

```diff
@@ -1072,7 +1078,10 @@ public void MoveToWorld(Point3D location, Map map)
 
             if (m_Map != null)
             {
-                m_Map.OnLeave(this);
+                if (!hadParent)
+                {
+                    m_Map.OnLeave(this);
```

Comment:
We shouldn't try to leave if there was already a parent, because that would be duplicative. This doesn't crash as duplicative cause of the broken logic in setting the Parent.